Source/WebKit/WebProcess/WebPage/WebCookieCache.cpp

```diff
@@ -132,4 +168,9 @@ NetworkStorageSession& WebCookieCache::inMemoryStorageSession()
 }
 #endif
 
+bool WebCookieCache::cacheContainsStaleCookies() const
```

Comment:
Maybe something like `cacheWillBeUpdatedSoon` Or `cacheMayBeOutOfSync`? `cacheContainsStaleCookies` sound like "some entries will be removed from it".

Reply:
I like `cacheMayBeOutOfSync()`, will update.